products/PDFmorph.rst

```diff
@@ -28,12 +28,9 @@ below.
 There are also a few other morphing transformations in the program.
 
 Finally, we note that PDFmorph should work on other spectra that are not PDFs, 
-though it has not been extensively texted beyond the PDF.
+though it has not been extensively tested beyond the PDF.
 
-PDFmorph will soon be included in :doc:`xPDFsuite <xPDFsuite>`, a powerful
-end-to-end software for X-ray PDF analysis. PxPDFsuite is designed for flexible,
-high throughput PDF analyses and contains many features for interrogating,
-comparing, and modeling data as well as simply doing data reduction.
+.. PDFmorph will soon be included in :doc:`xPDFsuite <xPDFsuite>`, a powerful end-to-end software for X-ray PDF analysis. PxPDFsuite is designed for flexible, high throughput PDF analyses and contains many features for interrogating, comparing, and modeling data as well as simply doing data reduction.
 
 .. figure:: ../images/pdfmorph_nacl_ex.png
 	:align: center
```

Comment:
It looks to me as if the morphing operations are being applied sequentially, in which case the label should be ``scale+smear`` and so on.    The colors on that plot are very ugly, as is the plot itself.  Is this using a good style sheet to make the plot?

Reply:
Also, here are some comments from below:
 - I think you want to activate the py3x env on windows, not py27
 - easu -> easy
 - are we keeping the easy_instal instructions or punting on that.  If punting, comment that one out.

Reply:
I'll look for replacements for the plot! It was just one of the plots I found on the PDFmorph github (other options [here](https://github.com/diffpy/diffpy.pdfmorph/tree/master/doc/images). Of those, it was the best/most comprehensive one I could find, but input is appreciated

Reply:
other stuff is fixed. Re: easy_install, Pavol said to keep it until we can do this kinda nuanced overhaul described [here](https://github.com/diffpy/diffpy.pdfmorph/issues/25)